starcheck/src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -28,7 +28,18 @@ use Inline Python => q{
 
 from chandra_aca.star_probs import acq_success_prob, prob_n_acq, mag_for_p_acq
 
+def _mag_for_p_acq(p_acq, date, t_ccd):
+    try:
+       date = date.decode()
```

Comment:
For these, if we're only Py3, I don't think I need to duck-type or even set a conditional on this.  date will always need a decode and it will always work.